Source/ModelGroups/Mesh_PrimitiveMode.cs

```diff
@@ -1,488 +1,356 @@
+                }),
+                CreateModel((properties, meshPrimitive) => {
+                    SetModePoints(properties, meshPrimitive);
+                    SetIndicesPoints(properties, meshPrimitive);
+                    SetIndicesComponentTypeInt(properties, meshPrimitive);
+                }),
+                CreateModel((properties, meshPrimitive) => {
+                    SetModeLines(properties, meshPrimitive);
+                    SetIndicesLines(properties, meshPrimitive);
+                    SetIndicesComponentTypeInt(properties, meshPrimitive);
+                }),
+                CreateModel((properties, meshPrimitive) => {
+                    SetModeLineLoop(properties, meshPrimitive);
+                    SetIndicesLineLoopOrTriangleFan(properties, meshPrimitive);
+                    SetIndicesComponentTypeInt(properties, meshPrimitive);
+                }),
+                CreateModel((properties, meshPrimitive) => {
+                    SetModeLineStrip(properties, meshPrimitive);
+                    SetIndicesLineStrip(properties, meshPrimitive);
+                    SetIndicesComponentTypeInt(properties, meshPrimitive);
+                }),
+                CreateModel((properties, meshPrimitive) => {
+                    SetModeTriangleStrip(properties, meshPrimitive);
+                    SetIndicesTriangleStrip(properties, meshPrimitive);
+                    SetIndicesComponentTypeInt(properties, meshPrimitive);
+                }),
+                CreateModel((properties, meshPrimitive) => {
+                    SetModeTriangleFan(properties, meshPrimitive);
+                    SetIndicesLineLoopOrTriangleFan(properties, meshPrimitive);
+                    SetIndicesComponentTypeInt(properties, meshPrimitive);
+                }),
+                CreateModel((properties, meshPrimitive) => {
+                    SetModeTriangles(properties, meshPrimitive);
+                    SetIndicesTriangles(properties, meshPrimitive);
+                    SetIndicesComponentTypeInt(properties, meshPrimitive);
+                }),
+                CreateModel((properties, meshPrimitive) => {
+                    SetModeTriangles(properties, meshPrimitive);
+                    SetIndicesTriangles(properties, meshPrimitive);
+                    SetIndicesComponentTypeByte(properties, meshPrimitive);
+                }),
+                CreateModel((properties, meshPrimitive) => {
+                    SetModeTriangles(properties, meshPrimitive);
+                    SetIndicesTriangles(properties, meshPrimitive);
+                    SetIndicesComponentTypeShort(properties, meshPrimitive);
+                }),
+            };
+
+            GenerateUsedPropertiesList();
         }
+    }
 
-        static internal class PointOnLine
+    static internal class PointOnLine
+    {
+        static public Vector3 FindPoint(Vector3 point1, Vector3 point2, float fractionOfSegment)
```

Comment:
Why put this function in a separate class? It would make more sense to me if this were a private static function of `Mesh_PrimitiveMode`, i.e. `private static Vector3 GetPointOnLine(...)`

Reply:
That makes sense to me, I'll make the change.

Reply:
Fixed